doc/ref_model/chapters/chapter03.rst

```diff
@@ -1806,7 +1806,7 @@ for their respective control functions. These APIs must have an ability to handl
 forwarding and control planes life cycle management will not be atomic. The offload functions that should be offered as
 services must have published and preferably standardized open APIs, but the application specific forwarding functions do
 not have to be open APIs since they will only communicate with the application tenant provided control functions.
-`P4 <https://p4.org/>`__ and `OpenConfig <https://openconfig.net/>`__ are examples of suitable languages and models,
```

Comment:
Is it clear why the linkcheck fails on this? https://openconfig.net/ opens in my browser nicely. Anyways I'm okay to change this to fix the builds, but would be nice to know.

Reply:
@CsatariGergely the certificate is clearly invalid - I'm hoping your web browser is warning u about such issues.

Reply:
@CsatariGergely It is a certificate issue and, thus, for example, it is blocked at MSFT on any browser. I also didn't work on my personal MAc - Safari.